### ISSUE TITLE
Where can I find the specification for .ps files?

### ISSUE THREAD

**Issue by projectoriented** [mrvollger/minimiro#3]
Hi Mitchell 👋 ,

Thanks for this! Do you know where I can find the .ps specification? I'd like to make some modifications to the drawing and hopefully can contribute to this a bit more in the future.

Thanks!

**Comment by mrvollger** [comment]
Hi @projectoriented,

All contributions are most welcome! 

ps stands for PostScript and there are a lot a various sources to learn about that. However, when I wrote this I worked off of and learned form the original Perl/PostScript file provided in the [miropeats](https://www.littlest.co.uk/software/bioinf/old_packages/miropeats/) publication. I wish I knew the language better so I could be of more help, but my additions to miropeats were also mostly by trial and error. 

Cheers,
Mitchell
